### PR TITLE
Sushant | MPOC-358 Update: Summary Bar IN OpenELIS

### PR DESCRIPTION
**Pull request by sushantpatil1214**

### CODE REVIEW

File: openelis/src/us/mn/state/health/lims/common/provider/query/PatientXmlCreator.java [hunk 1]
```diff
@@ -74,6 +89,11 @@ public void createXml(Patient patient, StringBuilder xml) {
 
         List<PatientIdentity> identityList = PatientUtil.getIdentityListForPatient(patient.getId());
 
+        Timestamp ts=patient.getBirthDate();  
+        Date date=ts;    
+	    SimpleDateFormat formatter = new SimpleDateFormat("dd/MM/yyyy");
```
Comment (krishnanspace) [inline openelis/src/us/mn/state/health/lims/common/provider/query/PatientXmlCreator.java]: Please correct the formatting

File: openelis/src/us/mn/state/health/lims/common/provider/query/PatientXmlCreator.java [hunk 2]
```diff
@@ -15,6 +15,21 @@
...
+import us.mn.state.health.lims.patientidentity.valueholder.PatientIdentity;
+import us.mn.state.health.lims.person.valueholder.Person;
+import java.sql.Timestamp;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import java.sql.Timestamp;
+import java.util.Calendar;
+import java.util.Set;
```
Comment (krishnanspace) [inline openelis/src/us/mn/state/health/lims/common/provider/query/PatientXmlCreator.java]: Please remove all the unused imports

File: openelis/src/us/mn/state/health/lims/common/provider/query/PatientXmlCreator.java [hunk 3]
```diff
@@ -41,7 +56,7 @@
 import java.util.List;
 
 public class PatientXmlCreator {
-
+
```
Comment (krishnanspace) [inline openelis/src/us/mn/state/health/lims/common/provider/query/PatientXmlCreator.java]: Remove this extra space

File: openelis/WebContent/pages/dashboard/dashboardMaster.jsp [hunk 4]
```diff
@@ -266,14 +270,26 @@ basePath = path + "/";
...
+
+        var splitPattern = dob.split("/");
+        if (splitPattern[0] == "xx")
+        {
+            
+            dateOfBirth= dateOfBirth.concat(" (Estimado)");
+
+        }
+
+
```
Comment (krishnanspace) [inline openelis/WebContent/pages/dashboard/dashboardMaster.jsp]: Remove these extra lines

File: openelis/src/MessageResources.properties [hunk 5]
```diff
@@ -2919,6 +2919,8 @@ dashboard.sample.column.link.collectSample = Collect Sample
 dashboard.sample.column.link.result = Result
 dashboard.sample.column.link.validate = Validate
 dashboard.sample.column.link.print = Print
+dashboard.Main.Contact = Main Contact
```
Comment (krishnanspace) [inline openelis/src/MessageResources.properties:2922]: Also remove these `^M` characters

Comment (sushantpatil1214) [reply openelis/src/MessageResources.properties:2922]: ^M is also reflected on Master branch of OpenElis , In local this not is reflected I think this By default. Please have look on master Repo.

**Pull request merged**